### PR TITLE
Somkape/test web app

### PR DESCRIPTION
**Pull request by SomkaPe**
Create dev sample web app for confidential client testing 
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/353

### CODE REVIEW

File: tests/dev apps/WebTestApp/WebApi/Startup.cs [hunk 1]
```diff
@@ -48,7 +45,7 @@ public Startup(IHostingEnvironment env)
                 .Build();
         }
 
-        public IConfigurationRoot Configuration { get; set; }
+        public static IConfigurationRoot Configuration { get; set; }
```
Comment (kpanwar) [inline tests/dev apps/WebTestApp/WebApi/Startup.cs]: does this need to be static?

File: tests/dev apps/WebTestApp/WebApp/Controllers/AccountController.cs [hunk 2]
```diff
@@ -46,9 +40,8 @@ public class AccountController : Controller
         public async Task Login(string returnUrl = null)
         {
             if (HttpContext.User == null || !HttpContext.User.Identity.IsAuthenticated)
-            {
-                await HttpContext.Authentication.ChallengeAsync(OpenIdConnectDefaults.AuthenticationScheme, new AuthenticationProperties { RedirectUri = "/" });
-            }
+                await HttpContext.Authentication.ChallengeAsync(OpenIdConnectDefaults.AuthenticationScheme,
```
Comment (kpanwar) [inline tests/dev apps/WebTestApp/WebApp/Controllers/AccountController.cs]: use curly braces when writing if statements

File: tests/dev apps/WebTestApp/WebApi/Startup.cs (hunk 1 above)
Comment (SomkaPe) [inline tests/dev apps/WebTestApp/WebApi/Startup.cs]: it is application wide config

File: tests/dev apps/WebTestApp/WebApp/Controllers/AccountController.cs (hunk 2 above)
Comment (SomkaPe) [inline tests/dev apps/WebTestApp/WebApp/Controllers/AccountController.cs]: Resharper did this, will change

**Pull request merged**